packages/cra-template/template/src/reportWebVitals.js

```diff
@@ -0,0 +1,13 @@
+const reportWebVitals = (onPerfEntry) => {
```

Comment:
This (and other changes) will also be needed in the TypeScript template :)

Reply:
Oops, added :)